src/pages/SourcePage.vue

```diff
@@ -148,4 +192,31 @@ export default SourcePage
 <!-- Add "scoped" attribute to limit CSS to this component only -->
 <style lang="scss" scoped>
 @import '../styles/text-only-page.scss';
+
+.table.is-bordered {
+  th {
+    cursor: pointer;
+  }
+
+  // th:focus,
```

Comment:
Why not just remove this instead of commenting it out?

Reply:
No reason; removed.